README.md

```diff
@@ -26,6 +26,13 @@ Document Server and Nextcloud Docker installation will install the preconfigured
     docker-compose up -d
     ```
 
+    To enable SSL encryption, create `certs` folder and copy the private key named as `tls.key` and the certificate named as `tls.crt` to it.  
```

Comment:
I don't have a lot of context about how this works. Where should the `certs` folder be created? And where can I get the `tls.key` and `tls.crt` to copy?

Reply:
in folder where docker-compose.yml and ssl.yml is

Reply:
Thank you. Where can I find the `tls.key` and `tls.cert`? Do I need to generate those some where? Do I need to get them from letsencrypt?

Reply:
yes, you can get them from lestencrypt for your domain.
if you do not have your own domain, you can generate a self-signed certificate.

Reply:
I've spent a few hours trying to get a `tls.key` and `tls.cert`. The letsencrypt website told me to use certbot, so I installed that, and ran `sudo certbot certonly --standalone`, which generated some `.pem` files. I don't see any of the files you mentioned anywhere. I can't even find them in the certbot docs: https://certbot.eff.org/docs/search.html?q=tls.crt

Can you tell me what needs to be done to get these files?

Reply:
put in `certs` `fullchain.pem` as `tls.crt` and `privkey.pem` as `tls.key`

Reply:
Amazing! It's working! <3 Thank you so much for your help!

For the sake of other people being able to follow the directions, it might be helpful to add a link to certbot directions and mention that the pem files should be renamed. I never would have guessed that.